scapy/layers/ntp.py

```diff
@@ -394,6 +394,9 @@ def getfield(self, pkt, s):
             extensions_len = len(extensions)
             while extensions_len >= 16:
                 ext_len = struct.unpack("!H", extensions[2:4])[0]
+                ext_len = min(ext_len, extensions_len)
+                if ext_len < 1:
+                    ext_len = extensions_len
```

Comment:
Perfect, but you've also add tests for VXLAN, which are out of the scope of this PR. Is that what you wanted or is that a mistake?